src/sentry/testutils/pytest/metrics.py

```diff
@@ -85,7 +85,7 @@ def new_create_snql_in_snuba(subscription, snuba_query, snql_query, entity_subsc
             is_performance_metrics = False
             is_metrics = False
             if isinstance(query.match, Entity):
-                is_performance_metrics = query.match.name.startswith("generic_metrics")
+                is_performance_metrics = query.match.name.startswith("generic")
```

Comment:
This is not the best fix but it was the easiest for the scope of this PR. We might want in the future to improve this.

Reply:
Out of curiosity, how does this change fix the tests?

Reply:
The acceptance tests were running into some endpoints which were querying generic metrics but via the entity key `generic_org_metrics_counters` which is still on generic metrics but failed the `generic_metrics` check. For this reason I relaxed the condition but a better solution would be to have some sort of mapping or inferring the use case id from the query and marking it as performance if it's not sessions.